azure-devops/run_build.yml

```diff
@@ -2,74 +2,75 @@
-  pool:
-    name: STL
+  - job: ${{ parameters.targetPlatform }}
+    pool:
+      name: STL
 
-  variables:
-    vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
-  steps:
-    - checkout: self
-      submodules: recursive
-    - task: PowerShell@2
-      displayName: 'Install MSVC Preview'
-      timeoutInMinutes: 30
-      # on "Hosted" agents only:
-      condition: or(contains(variables['Agent.Name'], 'Azure Pipelines'), contains(variables['Agent.Name'], 'Hosted Agent'))
-      inputs:
-        targetType: filePath
-        filePath: $(Build.SourcesDirectory)/azure-devops/install_msvc_preview.ps1
-    - task: Cache@2
-      displayName: Cache vcpkg
-      timeoutInMinutes: 10
-      inputs:
-        key: '"${{ parameters.targetPlatform }}" | $(Build.SourcesDirectory)/.git/modules/vcpkg/HEAD'
-        path: '$(vcpkgLocation)'
-        cacheHitVar: CACHE_RESTORED
-    - task: run-vcpkg@0
-      displayName: 'Run vcpkg to Install boost-build'
-      condition: and(ne(variables.CACHE_RESTORED, 'true'), contains('${{ parameters.targetPlatform }}', 'arm'))
-      timeoutInMinutes: 10
-      inputs:
-        vcpkgArguments: 'boost-build:x86-windows'
-        vcpkgDirectory: '$(vcpkgLocation)'
-    - task: run-vcpkg@0
-      displayName: 'Run vcpkg'
-      timeoutInMinutes: 10
-      inputs:
-        vcpkgArguments: 'boost-math'
-        vcpkgDirectory: '$(vcpkgLocation)'
-        vcpkgTriplet: ${{ parameters.targetPlatform }}-windows
-    - task: run-cmake@0
-      displayName: 'Build Support Tools'
-      timeoutInMinutes: 2
-      condition: eq('${{ parameters.targetPlatform }}', 'x64')
-      inputs:
-        cmakeListsTxtPath: 'tools/CMakeSettings.json'
-        useVcpkgToolchainFile: true
-        configurationRegexFilter: '.*x64-Release.*'
-        buildDirectory: $(Build.ArtifactStagingDirectory)/tools
-    - task: BatchScript@1
-      displayName: 'Enforce clang-format'
-      timeoutInMinutes: 60
-      condition: eq('${{ parameters.targetPlatform }}', 'x64')
-      inputs:
-        filename: 'azure-devops/enforce-clang-format.cmd'
-        failOnStandardError: true
-        arguments: '$(Build.ArtifactStagingDirectory)/tools/parallelize/parallelize.exe'
-    - task: BatchScript@1
-      displayName: 'Validate Files'
-      timeoutInMinutes: 2
-      condition: eq('${{ parameters.targetPlatform }}', 'x64')
-      inputs:
-        filename: 'azure-devops/validate-files.cmd'
-        failOnStandardError: true
-        arguments: '$(Build.ArtifactStagingDirectory)/tools/validate/validate.exe'
-    - task: run-cmake@0
-      displayName: 'Build the STL'
-      timeoutInMinutes: 10
-      inputs:
-        cmakeListsTxtPath: 'CMakeSettings.json'
-        useVcpkgToolchainFile: true
-        configurationRegexFilter: '.*${{ parameters.targetPlatform }}.*'
-        buildDirectory: $(Build.ArtifactStagingDirectory)/${{ parameters.targetPlatform }}
+    variables:
+      vcpkgLocation: "$(Build.SourcesDirectory)/vcpkg"

```

Comment:
What's with all of these "single-quote to double-quote" changes? They greatly obscure the actual functional changes. Am I correct that the addition of `clean: true` on line 13 is the only functional change?

Reply:
ugh it's the red-hat yaml extension for vscode.

Reply:
trying to be a little too clever.